src/libs/Notification/PushNotification/configureForegroundNotifications/index.android.js

```diff
@@ -0,0 +1,9 @@
+import Airship from '@ua/react-native-airship';
+import shouldShowPushNotification from '../shouldShowPushNotification';
+
+/**
+ * Configures notification handling while in the foreground on iOS and Android. This is a no-op on other platforms.
```

Comment:
Same here, we should update the comment as this is just for Android

Reply:
Just got rid of the android and iOS specific docs and left the root one to explain